StripeFinancialConnections/StripeFinancialConnections/Source/Native/NetworkingLinkSignupPane/NetworkingLinkSignupViewController.swift

```diff
@@ -272,10 +272,10 @@ final class NetworkingLinkSignupViewController: UIViewController {
     }
 }
 
-extension NetworkingLinkSignupViewController: NetworkingLinkSignupBodyFormViewDelegate {
+extension NetworkingLinkSignupViewController: LinkSignupFormViewDelegate {
```

Comment:
since this modifies a little bit of existing code, its worth also running `bitrise run financial-connections-stability-tests` as a double-check that things work as expected (I'd expect it to)